src/utils/validateStrings.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Checks if an empty string of any length has been passed in
+ * @param {string} itemName
+ * @returns {boolean}
+ */
+export const isEmpty = (itemName) =>
+	typeof itemName !== 'string' || itemName.trim().length === 0;
```

Comment:
Just for my own curiosity, when would the itemName's type not be a string?

Reply:
If it's undefined or null!

I believe we added this check after running into an issue from the itemName being undefined

Reply:
Oh gotcha! Yeah it's always good to do null checking :)

Reply:
solid solution here! This is a piece of functionality that would be a great example of one we could write a unit test for in upcoming weeks if you get done with your week's ticket ahead of time, and would be a great one to have in place to ensure things continue to work as expected as the app evolves! Not something that needs to be done today but definitely keep it in mind for future issues!